lib/puppet/provider/vcsrepo/cvs.rb

```diff
@@ -32,10 +32,10 @@ def working_copy_exists?
       end
     else
       directory = File.join(@resource.value(:path), 'CVSROOT')
-      return false if not File.directory?(directory)
+      return false unless File.directory?(directory) # TODO: Errors if set to unless
```

Comment:
Where are the errors? Should this be a rubocop TODO?

Reply:
remnant i forgot to remove